icons/influxdb/influxdb-original-wordmark.svg

```diff
@@ -0,0 +1 @@
6c1.246-.352 1.246-.934-.043-1.278Zm0 0"/><path style="fill-rule:nonzero;fill:#020a47;fill-opacity:1" d="M4.996 100.613c-.805 0-1.488-.261-2.047-.789a2.56 2.56 0 0 1-.812-1.914c0-.789.27-1.437.812-1.945.559-.524 1.242-.79 2.047-.79.805 0 1.477.266 2.02.79.543.508.812 1.156.812 1.945 0 .75-.27 1.39-.812 1.914-.543.528-1.215.79-2.02.79Zm-2.55 16.82v-14.538H7.52v14.539Zm8.488 0v-14.538h4.992l.058 1.804a3.903 3.903 0 0 1 1.653-1.664c.73-.39 1.57-.59 2.523-.59 1.63 0 2.863.528 3.703 1.578.86 1.032 1.29 2.555 1.29 4.563v8.848h-5.075v-7.891c0-1.707-.644-2.563-1.937-2.563-.692 0-1.223.247-1.598.735-.355.469-.531 1.144-.531 2.027v7.692Zm19.015 0V106.98h-2.328v-4.085h2.328v-1.407c0-2.234.543-3.965 1.63-5.183 1.1-1.223 2.651-1.832 4.651-1.832a8.91 8.91 0 0 1 1.63.14c.523.075.96.188 1.316.34l-.25 4.223a2.71 2.71 0 0 0-.758-.196 4.152 4.152 0 0 0-.785-.085c-.805 0-1.403.234-1.797.707-.375.468-.559 1.21-.559 2.222v1.07h3.422v4.086h-3.422v10.454Zm10.742 0v-22.82h5.079v22.82Zm13.274.45c-1.59 0-2.805-.508-3.645-1.524-.843-1.03-1.265-2.535-1.265-4.504v-8.96h5.105v7.918c0 .863.149 1.5.45 1.914.316.414.812.62 1.488.62.672 0 1.187-.234 1.539-.702.375-.489.562-1.176.562-2.06v-7.69h5.078v14.539h-4.992l-.058-1.805a3.844 3.844 0 0 1-1.684 1.66c-.727.395-1.586.594-2.578.594Zm11.613-.45 5.27-7.496-4.993-7.042h5.668l2.356 3.945 2.3-3.945h5.383l-4.964 7.101 5.3 7.438h-5.691l-2.637-4.395-2.61 4.395Zm25.211.45c-1.289 0-2.45-.32-3.477-.957-1.007-.64-1.804-1.524-2.382-2.649-.582-1.144-.871-2.441-.871-3.886 0-1.391.308-2.649.925-3.778.637-1.144 1.489-2.047 2.551-2.703a6.476 6.476 0 0 1 3.535-1.015 5.59 5.59 0 0 1 2.918.789c.899.527 1.617 1.27 2.16 2.226V94.613h2.383v22.82h-2.328l-.055-2.566c-.562.957-1.3 1.7-2.218 2.227-.914.527-1.961.789-3.14.789Zm.617-2.14c.973 0 1.824-.227 2.555-.677.727-.453 1.297-1.082 1.71-1.886.43-.809.645-1.739.645-2.79 0-1.054-.214-1.984-.644-2.788-.414-.809-.984-1.438-1.711-1.887-.73-.453-1.582-.68-2.555-.68-.953 0-1.804.227-2.55.68-.731.45-1.31 1.078-1.739 1.887-.414.804-.617 1.734-.617 2.789 0 1.05.203 1.98.617 2.789.43.804 1.008 1.433 1.738 1.886.747.45 1.598.676 2.551.676Zm19.637 2.14c-1.102 0-2.102-.262-3-.79-.879-.527-1.57-1.269-2.074-2.226l-.059 2.567h-2.328v-22.82h2.387v11.296c.558-.957 1.297-1.7 2.215-2.226.914-.524 1.96-.79 3.14-.79 1.309 0 2.469.32 3.477.961 1.011.637 1.804 1.52 2.386 2.645.579 1.129.868 2.426.868 3.89 0 1.387-.317 2.657-.953 3.801a7.31 7.31 0 0 1-2.524 2.707c-1.047.657-2.226.985-3.535.985Zm-.336-2.14c.973 0 1.824-.227 2.555-.677a4.706 4.706 0 0 0 1.71-1.886c.43-.809.641-1.739.641-2.79 0-1.054-.21-1.984-.64-2.788-.414-.809-.985-1.438-1.711-1.887-.73-.453-1.582-.68-2.555-.68-.953 0-1.805.227-2.55.68-.731.45-1.31 1.078-1.739 1.887-.414.804-.617 1.734-.617 2.789 0 1.05.203 1.98.617 2.789.43.804 1.008 1.433 1.738 1.886.746.45 1.598.676 2.551.676Zm10.328-17.009v-2.996h-1.11v-.593h2.9v.593h-1.102v2.996Zm2.406 0 .184-3.59h.664l1.09 2.282 1.113-2.281h.664l.184 3.59h-.61l-.113-2.657-.984 2.035h-.485l-.992-2.054-.11 2.675Zm0 0"/></svg>
```

Comment:
Is it possible to combine the paths here?

Reply:
I'll see if I can do that. Thanks. 🙂